web/static/sass/wtt.scss

```diff
@@ -901,4 +908,33 @@ address {
     margin-bottom: 1em;
 }
 
+$mysoc-footer-border-top-color: #E9E9E9;
+$mysoc-footer-background-color: $body-bg;
+$mysoc-footer-link-text-color: $link-color;
+$mysoc-footer-link-hover-text-color: #333;
+$mysoc-footer-donate-background-color: $content-background-colour;
+$mysoc-footer-donate-button-background-color: $success-color;
+
+@import "ms_footer";
+
+.mysoc-footer {
+  font-family: $header-font-family;
+  border-top-width: 1px;
```

Comment:
See https://github.com/mysociety/standard-footer/issues/1#issuecomment-215486076 for a potential alternative to setting this width separately.